headless-server-commerce-base-lib/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -1,2 +1,2 @@
 com.coremedia.blueprint.headlessserver.CommerceCaasAutoConfiguration
-com.coremedia.blueprint.headlessserver.CorsAutoConfiguration
+com.coremedia.cms.web_security.cors.CorsAutoConfiguration
```

Comment:
This import isn't necessary as `middle.coremedia-web-security` is on the classpath